CODEOWNERS

```diff
@@ -0,0 +1,75 @@
+# Scheduler
+/master/internal/scheduler/    @aaron276h @rb-determined-ai
+
+# Provisioner
+/master/internal/provisioner/  @shiyuann
+
+# Model registry
+/master/pkg/model              @eecsliu @katport
+/proto/pkg/modelv1             @eecsliu @katport
+
+# WebUI
+/webui/                        @hkang1 @hamidzr
```

Comment:
maybe we separate the sub-WebUI modules :thinking:

Reply:
Into what? And who would own which part?

Reply:
maybe a separate line for WebUI e2e tests? as-is also works.